src/coord/src/coord.rs

```diff
@@ -2187,13 +2188,17 @@ impl Coordinator {
                                 if let Some(tables) = &mut self.persisted_tables {
                                     tables.write(id, &updates);
                                 }
-                                self.broadcast(SequencedCommand::Insert { id, updates });
+
+                                updates_map.insert(id, updates);
```

Comment:
Does it make sense to check that `id` doesn't already exist in `updates_map`. I dunno if it's possible to have that in this context?